staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -601,7 +523,7 @@ func (r *Request) tryThrottleWithInfo(ctx context.Context, retryInfo string) err
 		// but we use a throttled logger to prevent spamming.
 		globalThrottledLogger.Infof("%s", message)
 	}
-	metrics.RateLimiterLatency.Observe(ctx, r.verb, r.finalURLTemplate(), latency)
+	metrics.RateLimiterLatency.Observe(ctx, r.verb, *r.URL(), latency)
```

Comment:
I have a feeling that this discussion already happened, but aren't we afraid of cardinality here?

Reply:
yeah, Damien replaced the metrics to only use host, path is no longer https://github.com/kubernetes/kubernetes/pull/106539 present

Reply:
So shouldn't we use host here too?

Reply:
the interface takes an url.URL() 

https://github.com/kubernetes/kubernetes/blob/343125cc6c4a8b83b6fad8778d9fc1cd7ad1f947/staging/src/k8s.io/client-go/tools/metrics/metrics.go#L40-L43

and is an exported interface, is ok to change the signature?

Reply:
aah ok - I misunderstood initially. So it's already fixed for this metric too already.

We can discuss changing the interface, but given it's a client one it's always better to avoid it (and even if not, it's not for this PR).

Reply:
:upside_down_face: